.github/workflows/test_examples.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       ZEPHYR_TOOLCHAIN_VARIANT: "zephyr"
       ZEPHYR_SDK_INSTALL_DIR: "/opt/zephyr-sdk"
       DEMO_HOME: "${{ github.workspace }}"
-      TENSORFLOW_PATH: "${{ github.workspace }}/tensorflow"
+      TENSORFLOW_PATH: "${{ github.workspace }}/tflite-micro"
```

Comment:
Actually, should we change the path? We have the "tensorflow" path in Arduino examples, so maybe we could keep it consistent.

Reply:
This pr changes the path https://github.com/antmicro/tensorflow-arduino-examples/pull/168